src/sdk/strategy/cbd-strategy.ts

```diff
@@ -78,7 +78,11 @@ export class DeployedCbdStrategy {
   ) {}
 
   public static create(dkgRitual: DkgRitual, porterUri: string) {
-    const decrypter = ThresholdDecrypter.create(porterUri, dkgRitual);
+    const decrypter = ThresholdDecrypter.create(
+      porterUri,
+      dkgRitual.id,
+      dkgRitual.dkgParams.threshold
```

Comment:
Is threshold still needed, even though it is now part of the ritual struct?

Reply:
The client needs to know how many shares to fetch from the nodes (`/cbd_decrypt` responses)

Reply:
Yes, but now if you have the ritual id, you can get the value of the `threshold` from the Coordinator agent when doing the retrieve, or am I misunderstanding.

Reply:
If you inspect the usage of `DeployedCbdStrategy.create` you can see that this is exactly what happens:
- https://github.com/nucypher/nucypher-ts/blob/taco-api/src/sdk/strategy/cbd-strategy.ts#L47-L48
- https://github.com/nucypher/nucypher-ts/blob/taco-api/src/sdk/strategy/cbd-strategy.ts#L95-L96

Reply:
Right 👍 . I mixed up my layers.

Side question: any reason we need the `DkgRitualParameters` object? This line could look nice as:
```typescript
    const decrypter = ThresholdDecrypter.create(
      porterUri,
      dkgRitual.id,
      dkgRitual.threshold
```
? Anyways, just something to think about.